src/java.base/windows/native/libnet/NetworkInterface.h

```diff
@@ -31,55 +31,10 @@
 /*
  * Structures used when enumerating interfaces and addresses
  */
-typedef struct _netaddr  {
-    SOCKETADDRESS    addr;                  /* IPv4 or IPv6 address */
-    SOCKETADDRESS    brdcast;
-    short            mask;
-    struct _netaddr *next;
+typedef struct _netaddr {
+    SOCKADDR_INET Address;
+    UINT8 PrefixLength;
+    struct _netaddr *Next;
 } netaddr;
 
```

Comment:
Can NetworkInterface.h be dropped, and netaddr moved to NetworkInterface.c?

Reply:
I did that in an earlier revision. Then @msheppar asked for it to be moved back. See https://github.com/openjdk/jdk/pull/12593#issuecomment-1434450339